operators/tokenizer/bert_tokenizer.cc

```diff
@@ -2,8 +2,8 @@
 
 #include <utility>
 
-BertTokenizerVocab::BertTokenizerVocab(std::string vocab) : raw_vocab_(vocab) {
-  auto tokens = SplitString(raw_vocab_, "\n", true);
+BertTokenizerVocab::BertTokenizerVocab(std::string_view vocab) : raw_vocab_(vocab) {
```

Comment:
is string_view introduced in C++ 17? IIRC, the largest C++ standard used in onnxruntime is c++14 for some reasons. And the onnxruntime-extensions may need to be consistent with it.

Reply:
@snnn , is C++ 17 already supported in onnxruntime now?